samples/sample_encode/src/sample_encode.cpp

```diff
@@ -122,7 +122,7 @@ void PrintHelp(msdk_char *strAppName, const msdk_char *strErrorMessage, ...)
 #if (MFX_VERSION >= 1027)
     msdk_printf(MSDK_STRING("   [-round_offset_in file]  - use this file to set per frame inter/intra rounding offset(for AVC only)\n"));
 #endif
-    msdk_printf(MSDK_STRING("   [-qsv-ff]                - Enable QuickSync Fixed Function (low-power HW) encoding mode\n"));
+    msdk_printf(MSDK_STRING("   [-lowpower:<on,off>]     - Turn this option ON to enable QuickSync Fixed Function (low-power HW) encoding mode\n"));
```

Comment:
I'm afraid it's not easy to "deprecate" the option. It's using in test system and some customers are used to it. I'd suggest to keep old one.

Reply:
@dmitryermilov i removed them only from "help", but left it handler.

Reply:
Alight